konfera/forms.py

```diff
@@ -0,0 +1,16 @@
+from django import forms
+from konfera.models import Speaker, Talk
+
+
+class SpeakerForm(forms.ModelForm):
+    class Meta:
+        model = Speaker
+        fields = '__all__'
```

Comment:
Please remove sponsor field, relation between sponsor and speaker will be set by us at this point.